R/shiny-recorder.R

```diff
@@ -215,6 +215,10 @@ RecordingSession <- R6::R6Class("RecordingSession",
         httpuv::interrupt()
         private$localServer <- NULL
         close(private$outputFile)
+        if (length(fileList)) {
+          fileList <- paste(sprintf("\t%s", private$postFiles), collapse = '\n')
+          message("Note: the following files containing uploaded data were created. They must be kept with the recording file.\n", fileList)
```

Comment:
`Notes: Uploaded files to keep with recording file:` ?

Reply:
(If not, capitalize the `t` in the first `the`.)

Reply:
Thanks, much better